assets/sass/modules/_admin-consent.scss

```diff
@@ -99,6 +99,9 @@ $image-size: 30px;
 
     input {
       width: 140px;
+      height: auto;
+      min-height: 50px;
```

Comment:
is `50px` the default height of the button?

Reply:
That value was derived from the _btns.scss file, that was the explicit set height of the button so to not modify the size, I reset it here as the min-height to ensure it stays the same when the text does not need to wrap.

Reply:
awesome sounds good